src/cli/import.rs

```diff
@@ -39,14 +39,27 @@ pub async fn init(
 	// Initialize opentelemetry and logging
 	crate::o11y::builder().with_log_level("error").init();
 
-	let root = Root {
-		username: &username,
-		password: &password,
+	let client = if username.is_none() {
+		connect(endpoint.to_owned()).await?
```

Comment:
`IntoEndpoint` is implemented for both `&str` and `&String` so we don't need to call `to_owned` here.